runtime/bindings/python/src/openvino/ie_api.py

```diff
@@ -35,22 +39,29 @@
 
 def normalize_inputs(py_dict: dict) -> dict:
     """Normalize a dictionary of inputs to contiguous numpy arrays."""
-    return {k: (np.ascontiguousarray(v) if isinstance(v, np.ndarray) else v)
+    return {k: (Tensor(v) if isinstance(v, np.ndarray) else v)
             for k, v in py_dict.items()}
 
 # flake8: noqa: D102
-def infer(request: InferRequest, inputs: dict = None) -> dict:
-    results = request._infer(inputs=normalize_inputs(inputs if inputs is not None else {}))
-    return {name: (blob.buffer.copy()) for name, blob in results.items()}
+def infer(request: InferRequest, inputs: dict = None) -> np.ndarray:
+    res = request._infer(inputs=normalize_inputs(inputs if inputs is not None else {}))
+    # Required to return list since np.ndarray forces all of tensors data to match in
+    # dimensions. This results in errors when running ops like variadic split.
+    return [copy.deepcopy(tensor.data) for tensor in res]
 
-# flake8: noqa: D102
-def get_result(request: InferRequest, name: str) -> np.ndarray:
-    return request.get_blob(name).buffer.copy()
+
+def infer_new_request(exec_net: ExecutableNetwork, inputs: dict = None) -> List[np.ndarray]:
+    res = exec_net._infer_new_request(inputs=normalize_inputs(inputs if inputs is not None else {}))
+    # Required to return list since np.ndarray forces all of tensors data to match in
+    # dimensions. This results in errors when running ops like variadic split.
+    return [copy.deepcopy(tensor.data) for tensor in res]
 
 # flake8: noqa: D102
-def async_infer(request: InferRequest, inputs: dict = None, userdata=None) -> None:  # type: ignore
-    request._async_infer(inputs=normalize_inputs(inputs if inputs is not None else {}),
-                         userdata=userdata)
+def start_async(request: InferRequest, inputs: dict = None, userdata = None) -> None:  # type: ignore
```

Comment:
remove ```InferRequest``` as input type and leave comment that request here maybe as AsyncInferQueue, as InferRequestWrapper